src/shared/api/urls.ts

```diff
@@ -208,7 +208,8 @@ export function getProxyUrlIfNecessary(url: any) {
 }
 
 export function getOncoKbApiUrl() {
-    return buildCBioPortalAPIUrl(`proxy/oncokb`);
+    // return buildCBioPortalAPIUrl(`proxy/oncokb`);
+    return buildCBioPortalAPIUrl(`proxy/monkeykb`);
```

Comment:
This is the tentative name of the new endpoint. We can change this one to a non human readable name as well.

Reply:
proxy/nothingToSeeHere